public/js/util/DevToolsUtils.js

```diff
@@ -0,0 +1,13 @@
+function reportException(who, exception) {
+  let msg = who + " threw an exception: ";
+  console.error(msg, exception);
+}
+
+function executeSoon(aFn) {
+  setTimeout(aFn, 0);
+};
```

Comment:
moved executeSoon over as I'm not sure if the `Services.tm` tick logic can be duplicated in a de-priveledged way. Also, this is fine for our purposes.